tests/integration_tests/build/test_licenses.py

```diff
@@ -0,0 +1,43 @@
+
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring codebase contains neccessary licenses."""
+
+import os
+
+import pytest
+
+CHROMIUM_COPYRIGHT = "Copyright 2017 The Chromium OS Authors. All rights reserved."
+CHROMIUM_LICENSE = "Use of this source code is governed by a BSD-style license that can be"
+AMAZON_COPYRIGHT = "Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved."
+AMAZON_LICENSE = "SPDX-License-Identifier: Apache-2.0"
+EXCLUDED_DIRECTORIES = ((os.path.join(os.getcwd(), 'build')))
```

Comment:
nit: I usually like to sort these alphabetically (if there's lots of variables it makes things easy to find), or separate them in groups with a blank line (e.g., the excluded directory is not used in the same way as the other 4 variable strings).